Your changes has been incorporated into v3 branch

techno-express opened the pull request:
Hi, 
Just to let you know, i have forked and added onto most of your work on ezSQL. It has now been added into an branch and eventually will be part of the main repo. 

I am one of the collaborators to the project now. I was in the process of write my own unit tests and other major rewrites, then i came upon your pull request, seen it closed with no reasons. A lot of great changes too.

sjstoelting commented:
Hey @techno-express ,
Thanks a lot for coming up with that.
As you may have seen, I haven't worked on this repository for some years.

Some of the tests I've done have to be done on new versions, there has been some behaviour changes of some of the RDBs in their latest releases.
My original intention to use ezSQL was because I had to handle some stuff running against Oracle DBs. I'm happy, that I don't have to do that anymore.

The merge into my branch is marked as complex, though, I have to do that locally.

Best regards
Stefanie

sjstoelting commented:
Hi @techno-express 
I merged the ezSQL master branch from upstream and switched the default branch to master.
In my branch ezSQL-3.0 I've added the information to use the original sources of ezSQL. In addition I published that as news on my website, I've seen several page accesses to my ezSQL news list.

Thanks for you work merging my stuff into ezSQL back again.

Stefanie

techno-express commented:
Thanks, I forgot to let you know of master branch is v3 now. That's goods
others still asking about the project. I thought i was the only one too see
the real benefit, and ease of use it brings.
I'm working on v4, will be awhile, but will be breaking changes, and more
interoperable with PSR-2, PSR-4, PSR--11 compliance. The project not going
dead on my watch.

On Thu, Oct 11, 2018 at 10:02 AM Stefanie Janine Stölting <
notifications@github.com> wrote:

> Hi @techno-express <https://github.com/techno-express>
> I merged the ezSQL master branch from upstream and switched the default
> branch to master.
> In my branch ezSQL-3.0 I've added the information to use the original
> sources of ezSQL. In addition I published that as news on my website, I've
> seen several page accesses to my ezSQL news list.
>
> Thanks for you work merging my stuff into ezSQL back again.
>
> Stefanie
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/sjstoelting/ezSQL3/pull/1#issuecomment-428962738>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AcZ6lV2YoySIhg1Qj0wIErWRFHgqnNbkks5uj096gaJpZM4S5Yun>
> .
>


-- 
Sincerely,
Lawrence Stubbs
